lib/renderer/api/desktop-capturer.js

```diff
@@ -31,8 +23,8 @@ const getSources = (options) => {
   return new Promise((resolve, reject) => {
     if (!isValid(options)) throw new Error('Invalid options')
 
-    const captureWindow = includes.call(options.types, 'window')
-    const captureScreen = includes.call(options.types, 'screen')
+    const captureWindow = options.types.includes('window')
```

Comment:
These are not equivalent. 

`[].includes.call({}, 'a') === false`
`{}.includes('a') === Error`

Reply:
isValid enforces that types is an array